cli/lsp/language_server.rs

```diff
@@ -217,6 +229,9 @@ pub struct Inner {
   maybe_testing_server: Option<testing::TestServer>,
   /// Services used for dealing with npm related functionality.
   npm: LspNpmServices,
+  /// This is moved out to its own task after initializing.
+  outside_lock_task_rx: Option<UnboundedReceiver<OutsideLockTaskFn>>,
+  outside_lock_task_tx: UnboundedSender<OutsideLockTaskFn>,
```

Comment:
This is very low level to be storing on `LanguageServer`. Maybe we could extract all this out to a separate struct and then provide higher level functions on that struct?